src/toolkit.c

```diff
@@ -1325,15 +1325,15 @@ void window_render( Window *w )
    /*
     * focused widget
     */
-   if (w->focus != -1) {
+   if (w->focus != -1){
       wgt = toolkit_getFocus( w );
       if ((wgt == NULL) || wgt_isFlag(wgt, WGT_FLAG_KILL))
          return;
       x  += wgt->x;
       y  += wgt->y;
       wid = wgt->w;
       hei = wgt->h;
-      toolkit_drawOutlineThick( x, y, wid, hei, 3, 4, &cGrey10, NULL );
+      toolkit_drawOutlineThick( x, y, wid, hei, 0, 2, (wgt->type == WIDGET_BUTTON ? &cGrey70 : &cGrey20), NULL );
```

Comment:
I don't really like the idea of having the focus outline change depending on the widget type. It would be better to try to be consistent I think.

Reply:
This "widget" concept doesn't really carry over into the user's perception of the interface.  A button needs to have additional prominence over something like an image/text widget.

Reply:
Most of this stuff doesn't really even need an outline, e.g., this list widget

![1607869347](https://user-images.githubusercontent.com/2954343/102014594-bfbec680-3d24-11eb-885b-0ce3890ea782.png)

I'd be just as happy removing that code for the general use case.

Reply:
If the list widget is focused, you can navigate through it, which isn't possible if it isn't though. So in that sense, it is useful though.